### PR TITLE
Update DemoBench documentation.

### PR DESCRIPTION
**Pull request by chrisr3**
Add download link and location of log file.

### CODE REVIEW

File: docs/source/demobench.rst [hunk 1]
```diff
@@ -6,11 +6,21 @@ DemoBench is a standalone desktop application that makes it easy to configure an
...
+
+.. _R3 website: https://www.corda.net/downloads
 
 Running DemoBench
 -----------------
 
+DemoBench writes a log file to the following location:
+
++--------------+-------------------------------------------+
+| MacOSX/Linux | ``$HOME/demobench/demobench.log``         |
```
Comment (mikehearn) [inline docs/source/demobench.rst]: Hm can we avoid cluttering the users home dir with stuff like that? We can use System.getProperty("java.io.tmpdir") instead.

Comment (chrisr3) [inline docs/source/demobench.rst]: Operating systems have a habit of thinking that temporary directories are "safe" to clear. I think a sub-directory of the user's home directory is a fine place to store the user's data. Isn't that precisely what it's for?

File: docs/source/demobench.rst [hunk 2]
```diff
@@ -6,11 +6,21 @@ DemoBench is a standalone desktop application that makes it easy to configure an
...
++--------------+-------------------------------------------+
+| Windows      | ``%USERPROFILE%\demobench\demobench.log`` |
++--------------+-------------------------------------------+
+
 Configuring a Node
   Each node must have a unique name to identify it to the network map service. DemoBench will also suggest local port numbers to use.
```
Comment (chrisr3) [inline docs/source/demobench.rst]: The uniqueness of the name _is_ enforced - I just thought I'd mention why :-).

I thought this paragraph was telling users what DemoBench is, and why they should use it:
> DemoBench is a standalone desktop application that makes it easy to configure and launch local Corda nodes. It is useful for training sessions, demos or just experimentation.

DemoBench doesn't include any "canned" demos yet. I think there's a separare JIRA task for that.

File: docs/source/demobench.rst [hunk 3]
```diff
@@ -6,11 +6,21 @@ DemoBench is a standalone desktop application that makes it easy to configure an
 Downloading
 -----------
 
-Installers compatible with the latest Corda release will soon be downloadable from the R3 website.
+Installers compatible with the latest Corda release can be downloaded from the `R3 website`_.
+
+.. _R3 website: https://www.corda.net/downloads
 
 Running DemoBench
 -----------------
```
Comment (chrisr3) [inline docs/source/demobench.rst]: Once you've downloaded and installed it, you run it and are immediately presented with the "Node Configuration" screen.

File: docs/source/demobench.rst [hunk 4]
```diff
@@ -6,13 +6,15 @@ DemoBench is a standalone desktop application that makes it easy to configure an
...
+.. _Corda website: https://www.corda.net/downloads
 
 Running DemoBench
 -----------------
 
 Configuring a Node
-  Each node must have a unique name to identify it to the network map service. DemoBench will also suggest local port numbers to use.
+  Each node must have a unique name to identify it to the network map service. DemoBench will suggest local port numbers to use.
 
   The first node will host the network map service, and therefore *must* be a notary. Hence only notary services will be available to be selected in the ``Services`` list. For subsequent nodes you may also select any of Corda's other built-in services.
```
Comment (gendal) [inline docs/source/demobench.rst]: Why does hosting the NMS imply you must be a notary?

Comment (chrisr3) [inline docs/source/demobench.rst]: It is due to the Explorer not displaying the node hosting the NMS unless that node is also a notary. While arguable a bug in Explorer, disallowing it prevents user confusion.

File: docs/source/demobench.rst [hunk 5]
```diff
@@ -6,15 +6,17 @@ DemoBench is a standalone desktop application that makes it easy to configure an
...
 
 Running DemoBench
 -----------------
 
 Configuring a Node
-  Each node must have a unique name to identify it to the network map service. DemoBench will also suggest local port numbers to use.
+  Each node must have a unique name to identify it to the network map service. DemoBench will suggest local port numbers to use.
 
-  The first node will host the network map service, and therefore *must* be a notary. Hence only notary services will be available to be selected in the ``Services`` list. For subsequent nodes you may also select any of Corda's other built-in services.
+  The first node will host the network map service, and we are forcing that node also to be a notary. Hence only notary services will be available to be selected in the ``Services`` list. For subsequent nodes you may also select any of Corda's other built-in services.
```
Comment (shamsasari) [inline docs/source/demobench.rst]: forcing? Why not just say "and we also make it the notary"

Comment (chrisr3) [inline docs/source/demobench.rst]: Because we are forcing it to be a notary by only offering notary services.

**Pull request merged**